src/ptrace/_UPT_reg_offset.c

```diff
@@ -709,6 +709,39 @@ const int _UPT_reg_offset[UNW_REG_LAST + 1] =
     [UNW_S390X_F14]     = 0x150,
     [UNW_S390X_F15]     = 0x150,
     [UNW_S390X_IP]      = 0x08
+#elif defined(UNW_TARGET_RISCV64)
```

Comment:
You may want to add riscv64 case in `ptrace/_UPT_get_dyn_info_list_addr.c` as well?
```sh
ptrace/_UPT_get_dyn_info_list_addr.c:83:3: warning: #warning Implement get_list_addr(), please. [-Wcpp]
   83 | # warning Implement get_list_addr(), please
```

Reply:
You may also want to add a riscv64 case in `coredump/_UCD_access_reg_linux.c` for coredump.

Reply:
For `_UPT_get_dyn_info_list_addr`, I don't see how I can get the address in a remote context (also see comment).

Reply:
> You may also want to add a riscv64 case in `coredump/_UCD_access_reg_linux.c` for coredump.

have same issue here when trying to build on Gentoo Linux, do you have any plan to support coredump? for build log it's here [1] if you are interested..
[1] https://dev.gentoo.org/~dlan/logs/libunwind_build.log

Reply:
Oops, forgot this one. Just pushed the simple fix and enabled coredump by default.

For `_UPT_get_dyn_info_list_addr`, I don't think it's possible and no port other than IA64 has implemented it. Marking this as resolved for now.